pkg/managers/pluginmanager/pluginmanager.go

```diff
@@ -137,14 +137,18 @@ func (p *PluginManager) Start(ctx context.Context) error {
 
 	g, ctx := errgroup.WithContext(ctx)
 
-	// run conntrack GC
-	ct, err := conntrack.New()
-	if err != nil {
-		return errors.Wrap(err, "failed to get conntrack instance")
+	// run conntrack GC only if packetparser is enabled
+	if p.isPluginEnabled("packetparser") {
```

Comment:
Thanks for refactoring this PR @manojks1999 

To improve efficiency, we can use `plugins` map  of `PluginManager` to perform an O(1) lookup. For example:

```go
// to ignore the first return value, which is the value associated with the key we can use _
// isPacketParserEnabled is a boolean that will be true if the key name exists in the map 
// and false otherwise.
_, isPacketParserEnabled := p.plugins["packetparser"]
```

For reference only, the plugin map gets populated here:
https://github.com/microsoft/retina/blob/e4427126cb97813d5267a0f4618931447606aeb7/pkg/managers/pluginmanager/pluginmanager.go#L61-L67